src/events/cd-event-ticket.vue

```diff
@@ -1,14 +1,19 @@
 <template>
-  <div class="cd-event-tickets">
-    <div class="cd-event-tickets__head"></div>
-    <div class="cd-event-tickets__ticket">
-      <span class="cd-event-tickets__name"><span class="cd-event-tickets__name-for">Name:</span>{{ user.firstName }} {{ user.lastName }}</span>
-      <div class="cd-event-tickets__ticket-selector"> 
-        <multiselect v-model="tickets" :options="ticketsOptions" group-label="name" group-values="tickets" :multiple="true" :searchable="false" :group-select="false" :placeholder="$t('Select tickets')" track-by="id" label="name" @close="onBlur" @open="onFocus" :data-vv-name="`tickets-${user.id}`" v-validate="'required'"></multiselect>
+  <div class="cd-event-tickets" :class="{ 'before-disabled': notAttending }">
```

Comment:
This should be following [BEM](http://getbem.com/introduction/) by using a modifier instead of its own class.

So the class being added should be `cd-event-tickets--disabled`.

Then below in your LESS, you can do something like

```
.cd-event-tickets {
  // existing styles

  &--disabled {
    &:before {
      background-image: radial-gradient(circle at .2% 25%, @cd-white .4em, transparent .5em), radial-gradient(circle at .2% 25%, #a9a9a9 .6em,transparent .7em);
    }

    .cd-event-tickets__head {
      background-color: #d3d3d3;
    }
  }
}
```

Reply:
Gotcha, thanks